llarp/context.cpp

```diff
@@ -60,20 +60,11 @@ namespace llarp
     }
 
     // Router config
-    if(!singleThreaded && config->router.workerThreads > 0 && !worker)
-    {
-      worker.reset(
-          llarp_init_threadpool(config->router.workerThreads, "llarp-worker"));
-    }
+    if(config->router.workerThreads <= 0)
+      config->router.workerThreads = 1;
 
-    if(singleThreaded)
-    {
-      num_nethreads = 0;
-    }
-    else
-    {
-      num_nethreads = config->router.num_nethreads;
-    }
+    worker = std::make_shared< llarp::thread::ThreadPool >(
+        config->router.workerThreads, 1024, "llarp-worker");
```

Comment:
hard-coded value 1024, what does it mean and should it be a named constant or config option?